brave/brave-core
# Update adaptive icons for Android

## Resolves https://github.com/brave/brave-browser/issues/27937

Updates Brave adaptive icons for Android.
New icons will appear slightly bigger.

<img width="536" alt="Screenshot 2023-01-26 at 4 12 11 PM" src="https://user-images.githubusercontent.com/3308503/214873200-a8da2614-85e0-4143-8836-f7885325ede0.png">

<img width="511" alt="Update_adaptive_icons_for_Android_by_simoarpe_·_Pull_Request__16875_·_brave_brave-core" src="https://user-images.githubusercontent.com/3308503/214874651-d05319a5-3dff-4ff1-97b4-7bf3041af835.png">

## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

   1. Launch Brave
   2. During onboarding, tap on `Set as default browser`
   3. Tap through to see and set the system-default browser
   4. Compare Brave's icon to the other browsers'
